CMP1903_A1_2324/Game.cs

```diff
@@ -11,13 +11,13 @@ internal class Game
     {
```

Comment:
Can simplify the 'die1, die2, die3', by using a for loop.
This would shorten the code and make it easier to understand